### PR TITLE
OC-12889 Common Event that the user has no access to is still displayed on Participant Details Page

### PR DESCRIPTION
**Pull request by rooney**
https://jira.openclinica.com/browse/OC-12889

reverts https://github.com/OpenClinica/OpenClinica/pull/2451/files

### CODE REVIEW

**Comment by svadla-oc** [comment]
👍

File: src/main/java/core/org/akaza/openclinica/dao/extract/OdmExtractDAO.java [hunk 1]
```diff
@@ -3400,7 +3400,7 @@ protected String studyEventAndFormMetaCondition(int parentStudyId, int studyId,
                                                     boolean isIncludedSite, String[] showArchived) {
         return " where sed.study_id = " + parentStudyId + " " + showArchived[0] + " and "
                 + this.getEventDefinitionCrfCondition(studyId, parentStudyId, isIncludedSite) + " " + showArchived[1] + " and edc.crf_id = crf.crf_id "
-                + showArchived[2] + " and crf.crf_id = cv.crf_id  " + showArchived[3]
+                + showArchived[2] + " and crf.crf_id = cv.crf_id  " + showArchived[3] + " and (sed.type != 'common' or pt.permission_tag_id is null or pt.permission_tag_id in (" + permissionTags + ")) "
```
Comment (svadla-oc) [inline src/main/java/core/org/akaza/openclinica/dao/extract/OdmExtractDAO.java:3403]: can you please verify that OC-11536 still works after this fix?

Comment (rooney) [reply src/main/java/core/org/akaza/openclinica/dao/extract/OdmExtractDAO.java:3403]: this commit: 34ad589d8595aa4459e614b285ad89d33fbfb044
combined with this one
cause the codebase to become uncompileable
fixing

Comment (rooney) [reply src/main/java/core/org/akaza/openclinica/dao/extract/OdmExtractDAO.java]: @svadla-oc now verified:
https://jira.openclinica.com/browse/OC-12889?focusedCommentId=205916&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-205916

Comment (rooney) [reply src/main/java/core/org/akaza/openclinica/dao/extract/OdmExtractDAO.java]: Hi @svadla-oc is there any further change needed?

**Comment by svadla-oc** [comment]
👍

**Pull request merged**